### PR TITLE
Update dead youtube link

### PR DESCRIPTION
**Pull request by thenetworkgrinch**
Zero to robot step 1 contained a private video on weidmuller connections. I replaced the video with a PNG of the weidmuller connector and modified the warning. The image was taken from [frc electrical bible](https://mililanirobotics.gitbooks.io/frc-electrical-bible/content/roboRIO/general_roborio_overview.html ).

### CODE REVIEW

**Comment by sciencewhiz** [comment]
What's the license/copyright status/source of the image? I couldn't find anything on the frc electrical bible

**Comment by thenetworkgrinch** [comment]
That took forever to find, but I FOUND IT! It seems to be a pull of another unavailable video from Weidmuller, however their USA division has this video which seems to be similar.
https://www.youtube.com/watch?v=Zsnro-9_ZfQ

Unavailable video = https://www.youtube.com/watch?v=Gl_3rqMspVI

Discovery link: https://www.rhopointcomponents.com/product/connectors-enclosures/omnimate-signal/pcb-signal-terminals/weidmuller-lsf-smt-5-08-90-series/

**Comment by thenetworkgrinch** [comment]
Should I update the image with a link to that video come to think of it? @sciencewhiz

**Comment by thenetworkgrinch** [comment]
Updated it with the original video because I couldn't find the license information from the video.

File: source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst [hunk 1]
```diff
@@ -170,7 +170,7 @@ Weidmuller Connectors
         <iframe src="https://www.youtube-nocookie.com/embed/Zsnro-9_ZfQ" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
-.. warning:: The strip length is ~5/16" (~8 mm)
+.. warning:: The reccommended wire strip length is ~5/16" (~8 mm).
```
Comment (PeterJohnson) [inline source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst:173]:
```suggestion
.. warning:: The recommended wire strip length is ~5/16" (~8 mm).
```

**Review by PeterJohnson — CHANGES_REQUESTED** [review]

**Comment by thenetworkgrinch** [comment]
Thanks! I committed your suggestion.

File: source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst (hunk 1 above)
Comment (thenetworkgrinch) [reply source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst]: Committed this suggestion 👍

File: source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst [hunk 2]
```diff
@@ -167,11 +167,10 @@ Weidmuller Connectors
 .. raw:: html
 
     <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe src="https://www.youtube-nocookie.com/embed/kCcDw3lDYis" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+        <iframe src="https://www.youtube-nocookie.com/embed/Zsnro-9_ZfQ" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
-
-.. warning:: The correct strip length is ~5/16" (~8 mm), not the 5/8" (~16 mm) mentioned in the video.
+.. warning:: The recommended wire strip length is ~5/16" (~8 mm).
```
Comment (sciencewhiz) [inline source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst:173]: I think the warning could be deleted. It was needed when the video gave the wrong value. However, the correct value is given below.

Comment (thenetworkgrinch) [reply source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst]: Just updated it to remove the warning

**Review by sciencewhiz — APPROVED** [review]

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**